packages/react-next/src/components/ContextualMenu/ContextualMenu.classNames.ts

```diff
@@ -0,0 +1,258 @@
+import { getDividerClassNames } from 'office-ui-fabric-react/src/components/Divider/VerticalDivider.classNames';
+import { getMenuItemStyles } from './ContextualMenu.cnstyles';
+import { ITheme, mergeStyleSets, getGlobalClassNames, getScreenSelector, ScreenWidthMaxMedium } from '../../Styling';
+import { IVerticalDividerClassNames } from 'office-ui-fabric-react/src/components/Divider/VerticalDivider.types';
```

Comment:
I think this is where the error is from:
```suggestion
import { getDividerClassNames } from 'office-ui-fabric-react/lib/components/Divider/VerticalDivider.classNames';
import { getMenuItemStyles } from './ContextualMenu.cnstyles';
import { ITheme, mergeStyleSets, getGlobalClassNames, getScreenSelector, ScreenWidthMaxMedium } from '../../Styling';
import { IVerticalDividerClassNames } from 'office-ui-fabric-react/lib/components/Divider/VerticalDivider.types';
```